### DISCUSSION TITLE
New Issue Templates

### DISCUSSION THREAD

**Discussion by NotMyFault** [IntellectualSites/FastAsyncWorldEdit#937] [General]
FastAsyncWorldEdit is now a part of the GitHub issue forms alpha, and we've set up a [new form for bug reports](https://github.com/IntellectualSites/FastAsyncWorldEdit/issues/new?assignees=&labels=Requires+Testing&template=bug_report.yml). This should hopefully make it easier both for people to report bugs and for us to test and replicate bugs.

Feel free to reply to this thread with any feedback about the form!

**Comment by QuickWrite** [comment]
I think it would make sense if there is a general issue template so that people can easily create an issue that is not restricted for whatever it may be in the future.
So that when there is a very specific case people don't need to go the extra mile and just can create an issue based on the template that just has some HTML comments or so.
